rdkit/Chem/Pharm3D/UnitTestEmbed.py

```diff
@@ -9,31 +9,41 @@
 #  of the RDKit source tree.
 #
 from __future__ import print_function
-from rdkit import RDConfig
-import unittest, sys, os
+
+import doctest
+import gzip
 import io
-from rdkit.six import PY3
-from rdkit.six.moves import cPickle
+import os
+import unittest
+
 from rdkit import Chem
-from rdkit.Chem import ChemicalFeatures, rdDistGeom
-from rdkit.Chem.Pharm3D import EmbedLib
-import gzip
 from rdkit import DistanceGeometry as DG
 from rdkit import Geometry
-import Pharmacophore
-import numpy
+from rdkit import RDConfig
+from rdkit.Chem import ChemicalFeatures, rdDistGeom
+from rdkit.Chem.Pharm3D import EmbedLib
+from rdkit.Chem.Pharm3D import Pharmacophore
+from rdkit.six import PY3
+from rdkit.six.moves import cPickle
 
 
 def feq(n1, n2, tol=1e-5):
   return abs(n1 - n2) <= tol
 
 
+def load_tests(loader, tests, ignore):  # pylint: disable=unused-argument
+  """ Add the Doctests from the module """
+  tests.addTests(
+    doctest.DocTestSuite(EmbedLib, optionflags=doctest.ELLIPSIS + doctest.NORMALIZE_WHITESPACE))
+  return tests
+
+
 class TestCase(unittest.TestCase):
 
   def setUp(self):
     self.dataDir = os.path.join(RDConfig.RDCodeDir, 'Chem/Pharm3D/test_data')
-    self.fdefBlock = \
-                   """DefineFeature HAcceptor1 [N,O;H0]
+    self.fdefBlock = """
```

Comment:
You're right. I give it a positive spin though. We also test that the parser is capable of handling extra lines ;-)